.template/spec/variants/web/dangerfile_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe 'Web variant - Dangerfile' do
+  subject { file('Dangerfile') }
+
+  it 'adds slim_lint lint' do
+    expect(subject).to contain('slim_lint.lint')
```

Comment:
We didn't add [slim](https://github.com/slim-template/slim) template engine in this template, should we add `slim_lint`?
I think if we keep it, we should add slim also

Reply:
I saw nimble projects using slim_lint so I added that. As the template doesn't have slim so we will get rid of that!

Reply:
Updated!

Reply:
@hoangmirs @rafayetn  How about creating a new issue to `adding slim and slim_lint` to the template and we can work on that later

Reply:
@anduonghien created [this issue]( https://github.com/nimblehq/rails-templates/issues/213) to add slim.